MAUI/DataForm/explicit-data-editors.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: post
+title: Explicit data editors of .NET MAUI DataForm control | Syncfusion
+description: Learn about the explicit data editors of Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+ 
+# Explicit data editors .NET MAUI DataForm (SfDataForm)
+
+## Explicitly create data editors
+
+The data form auto generates the editors based on the data type, you can explicitly add the data editors by adding [SfDataForm.Items](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_Items) manually and you need to change the [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) property to false.
```

Comment:
The data form auto-generates the editors based on the data type. You can explicitly add the data editors by adding [SfDataForm.Items](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_Items) manually, and you need to change the [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) property to false.

Reply:
Modified the content